backend/src/main/java/de/neuefische/backend/service/MarketService.java

```diff
@@ -0,0 +1,60 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.model.Market;
+import de.neuefische.backend.repository.MarketRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+@Service
+@RequiredArgsConstructor
+public class MarketService {
+    private final MarketRepository marketRepository;
+
+    public List<Market> getAllMarkets() {
+        return marketRepository.findAll();
+    }
+
+    public Market addNewMarket(String marketName) {
+        Market market = Market.builder()
+                .marketId(null)
```

Comment:
When using the builder pattern, you dont need to explicity set the id. It will set to null automatilcy